idaplugin/rematch/network.py

```diff
@@ -103,6 +103,8 @@ def query(method, url, server=None, token=None, params=None, json=False):
     logger('network').info("[response] {}".format(return_obj))
     return return_obj
   except Exception as ex:
+    import traceback
+    logger.network('network').error(traceback.print_exc())
```

Comment:
`print_exc` returns `None` and prints the traceback to stdout. I think that `format_exc` does what you want here.

Also, I think the code that writes the trackback will fit better inside exceptions.py, but i'm not sure about it. what do you think?